docs/ocis/development/build.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Build"
+date: 2020-02-27T20:35:00+01:00
+weight: 30
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/development
+geekdocFilePath: build.md
+---
+
+{{< toc >}}
+
+## Build requirements
+
+see [Development - Getting Started]({{< relref "getting-started.md/#requirements">}})
+
+## Get the sources
+
+{{< highlight txt >}}
+git clone https://github.com/owncloud/ocis.git
+cd ocis
+{{< / highlight >}}
+
+## Build the oCIS binary
+
+The oCIS binary source is in the `ocis` folder inside the oCIS repository. In this folder you can build the oCIS binary:
+
+{{< highlight txt >}}
+make generate
+make build
```

Comment:
```suggestion
cd ocis
make generate
make build
```

This would make it really clear to generate-build from inside the `ocis` folder of the `ocis` repo. (The `cd ocis` after `git clone` above at line 20 just gets you into the root of the repo)